### DISCUSSION TITLE
Request headers to stitched GraphQL API endpoints

### DISCUSSION THREAD

**Discussion by lightstrike** [hasura/graphql-engine#7216] [General]
Does Hasura include URIs in request headers to stitched GraphQL API endpoints? We're specifically looking for when Hasura acts as proxy to WPGraphQL requests. We want to pass a custom header that will be used along with the URI value on the WPGraphQL side.

**Comment by dsandip** [comment]
@lightstrike Could you please provide an example of the URIs you are looking to include in the request to your remote schema? Also, please check out this section on adding custom headers in the request to your remote schema: https://hasura.io/docs/latest/graphql/core/remote-schemas/adding-schema.html#step-2-merge-remote-schema. Does this work for your use-case?

**Reply by lightstrike** [reply]
Hi @dsandip, we want to include the origin of the request to Hasura, ie. if we made a request from www.example.com/shop, that URI would be included as a header for the stitched GraphQL service to utilize in a mutation.

**Reply by motleydev** [reply]
@lightstrike It should be possible to cofigure per this documentation. Can you confirm if this solves the problem for you? https://hasura.io/docs/latest/graphql/core/remote-schemas/auth/remote-schema-forward-auth.html#passing-headers-from-hasura-to-your-remote-schema
